Dockerfile

```diff
@@ -763,6 +802,9 @@ ARG ENABLE_FDKAAC=
 # large things on the stack.
 RUN wget $WGET_OPTS -O ffmpeg.tar.bz2 "$FFMPEG_URL"
 RUN echo "$FFMPEG_SHA256  ffmpeg.tar.bz2" | sha256sum --status -c -
+# Had to add extra-libs="-lstdc++" to workaround an issue where when ffmpeg checks the availability of
+# libjxl, it tries to build a test program with gcc instead of g++ and it will fail due to missing 
+# libstdc++ symbols.
```

Comment:
Had to do the same for libvmaf but instead patch the pkgconfig https://github.com/wader/static-ffmpeg/blob/master/Dockerfile#L83 don't know which is a best really :)

Do you know if the has been reported upstream? could add some link to issue if so

Reply:
I am unsure this kind of "bug" should be reported to which side, since there are 2 approaches to workaround them while we can consider either the library should ship .pc with stdc++ dependency or ffmpeg should switch to g++ to test library availability for those cases.

Reply:
Yes true it's a bit unclear, as long as it links and works i'm good 👍

Reply:
This comment is not needed now i guess? or add more context to comment at libjxl build?